network/src/commonMain/kotlin/com/wire/kalium/network/api/v0/authenticated/AssetApiV0.kt

```diff
@@ -91,26 +90,30 @@ internal open class AssetApiV0 internal constructor(
      * Build path for assets endpoint download.
      * The case for using V3 is a fallback and should not happen.
      */
-    protected open fun buildAssetsPath(assetId: AssetId): String {
-        return if (assetId.domain.isNotBlank()) "$PATH_PUBLIC_ASSETS_V4/${assetId.domain}/${assetId.value}"
-        else "$PATH_PUBLIC_ASSETS_V3/${assetId.value}"
+    protected open fun buildAssetsPath(assetId: String, assetDomain: String?): String {
+        if (assetDomain?.isBlank() == true) {
+            throw Error("Asset domain is not set")
```

Comment:
Can we add a test for this?

Reply:
this is already tested by asserting that the past parameters are correct